src/logger.ts

```diff
@@ -0,0 +1,25 @@
+import winston from 'winston';
+
+const logger = winston.createLogger({
+    level: 'info',
+    format: winston.format.combine(
+        winston.format.timestamp(),
+        winston.format.json()
+    ),
+    transports: [
+        new winston.transports.File({
+            filename: 'combined.log',
```

Comment:
```suggestion
            filename: minimist(process.argv.slice(2)).logLocation || 'combined.log',
```
If you want simple, but not that hardcoded?

Reply:
Good idea.